documentation/src/main/asciidoc/spatial.asciidoc

```diff
@@ -314,13 +314,8 @@ The default (field) name in case `@Spatial` is placed on the entity level is `or
 [NOTE]
 .Distance projection and null values
 ====
-Using distance projection on non `@Spatial` enabled entities and/or with a non spatial Query will have
-unexpected results as entities not spatially indexed and/or having `null` values for latitude or
-longitude will be considered to be at (0,0)/(lat,0)/(0,long).
-
-Using distance projection with a spatial query on spatially indexed entities having, eventually,
-`null` values for latitude and/or longitude is safe as they will not be found by the spatial query
-and won't have distance calculated.
+When a spatial field on an entity has a `null` value for either its latitude or longitude (or both),
+the resulting projected distance will always be `Double.MAX_VALUE` (`1.7976931348623157e+308`).
```

Comment:
Maybe... But we'd have to convert the `Double.MAX_VALUE` returned by Elasticsearch to `null`, in that case. I'll see if it can be done reliably. One thing that bothers me is I'm not sure I will be able to do the same with sorts (consider the value is null), and I'd rather have the projection and sorts be consistent.